### PR TITLE
Fix exception super

### LINKED ISSUE #2273: Grape:Exceptions::Base missing super?
Hello,

While looking at some exceptions in our logs, I found that the attribute `message` wasn't shown when doing `to_s`. For instance,
```ruby
e = Grape::Exceptions::EmptyMessageBody.new('my message')
e.to_s =  "Grape::Exceptions::EmptyMessageBody"
```
When reading about [Exception](https://ruby-doc.org/core-2.5.1/Exception.html)
```
to_s → string
Returns exception's message (or the name of the exception if no message is set).
```

If we add `super(message)` in missing in Grape::Exceptions::Base, it works as expected

I'll open a PR

Thanks.

### PR DESCRIPTION
**Pull request by ericproulx**
Fix  #2273

### CODE REVIEW

File: lib/grape/exceptions/base.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ def initialize(status: nil, message: nil, headers: nil, **_options)
         @status  = status
         @message = message
         @headers = headers
+        super(message)
```
Comment (dblock) [inline lib/grape/exceptions/base.rb:16]: Does this duplicate storing `message`, i.e. is `super.to_s` or something of the kind provides the property and we can remove `@message` from our implementation?

Comment (ericproulx) [reply lib/grape/exceptions/base.rb:16]: Makes total sense

File: lib/grape/exceptions/base.rb [hunk 2]
```diff
@@ -7,12 +7,12 @@ class Base < StandardError
       BASE_ATTRIBUTES_KEY = 'grape.errors.attributes'
       FALLBACK_LOCALE = :en
 
-      attr_reader :status, :message, :headers
```
Comment (dblock) [inline lib/grape/exceptions/base.rb:10]: This is a breaking change like so, meaning that I cannot do .message on an exception, right? Let's reimplement `def message` that calls `to_s`?

**Review by dblock — CHANGES_REQUESTED** [review]

File: lib/grape/exceptions/base.rb (hunk 2 above)
Comment (ericproulx) [reply lib/grape/exceptions/base.rb:10]: `message` exists through inheritance from StandardError

Comment (ericproulx) [reply lib/grape/exceptions/base.rb:10]: I've just pushed a `spec` for `message`

**Review by dblock — APPROVED** [review]

File: lib/grape/exceptions/base.rb [hunk 3]
```diff
@@ -7,12 +7,12 @@ class Base < StandardError
...
+      attr_reader :status, :headers
 
       def initialize(status: nil, message: nil, headers: nil, **_options)
         @status  = status
-        @message = message
         @headers = headers
+        super(message)
```
Comment (kuraga) [inline lib/grape/exceptions/base.rb:15]: Should be two lines above :)

Comment (dblock) [reply lib/grape/exceptions/base.rb:15]: Does it matter? If you think it does, PR?

Comment (kuraga) [reply lib/grape/exceptions/base.rb:15]: #2304

**Pull request merged**